### PR TITLE
refactor: updated monitoring related docs

### PR DESCRIPTION
**Pull request by ffforest**
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):

- [ ] N
- [ ] Y 

#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Other

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [ ] N
- [ ] Y 

#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

- [ ] Unit test
- [ ] Integration test
- [ ] Benchmark (add benchmark stats below)
- [ ] Manual test (add detailed scripts or steps below)
- [ ] Other

#### 6. Release note

Please refer to [Release Notes Language Style Guide](https://kusionstack.io/docs/governance/release-policy/) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: docs/kusion/4-configuration-walkthrough/8-monitoring.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 ---
-id: monitoring
+sidebar_position: 8
```
Comment (healthjyk) [inline docs/kusion/4-configuration-walkthrough/8-monitoring.md:2]: no-need for the sidebar-position

File: docs/kusion/4-configuration-walkthrough/8-monitoring.md [hunk 2]
```diff
@@ -26,93 +26,62 @@ import catalog.models.schema.v1.workload as wl
 import catalog.models.schema.v1.monitoring as m
 ```
 
-## Project-level configurations
+## Workspace configurations
```
Comment (healthjyk) [inline docs/kusion/4-configuration-walkthrough/8-monitoring.md:29]: Do not include workspace configuration, only contain configuration for app developer

Comment (healthjyk) [inline docs/kusion/4-configuration-walkthrough/8-monitoring.md:29]: Do not include workspace configuration, only contain configuration for app developer

File: docs/kusion/6-reference/2-modules/2-workspace-configs/_category_.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+  "label": "Workspace Configs",
+  "position": 2
```
Comment (healthjyk) [inline docs/kusion/6-reference/2-modules/2-workspace-configs/_category_.json:3]: no-need for the position

File: docs/kusion/6-reference/2-modules/1-catalog-models/_category_.json [hunk 4]
```diff
@@ -0,0 +1,4 @@
+{
+  "label": "Catalog Models",
+  "position": 1
```
Comment (healthjyk) [inline docs/kusion/6-reference/2-modules/1-catalog-models/_category_.json:3]: no-need for the position

File: docs/kusion/4-configuration-walkthrough/8-monitoring.md (hunk 1 above)
Comment (ffforest) [reply docs/kusion/4-configuration-walkthrough/8-monitoring.md]: done

File: docs/kusion/4-configuration-walkthrough/8-monitoring.md (hunk 2 above)
Comment (ffforest) [reply docs/kusion/4-configuration-walkthrough/8-monitoring.md:25]: done

File: docs/kusion/6-reference/2-modules/2-workspace-configs/_category_.json (hunk 3 above)
Comment (ffforest) [reply docs/kusion/6-reference/2-modules/2-workspace-configs/_category_.json:3]: done

File: docs/kusion/4-configuration-walkthrough/8-monitoring.md (hunk 2 above)
Comment (ffforest) [reply docs/kusion/4-configuration-walkthrough/8-monitoring.md:25]: done

**Review by healthjyk — APPROVED** [review]

**Pull request merged**